en/components/for_of.md

```diff
@@ -200,6 +200,12 @@ The following code snippet demonstrates how to use the `even` property in an `ng
 ```
 In the example above, an `even` class is assigned to every even div element.
 
+## Known Limitations
+
+|Limitation|Description|
+|--- |--- |
+| Scroll position of components that use igxForOf is not preserved |When changing the context of an igxForOf-using element, the scrollbar position won't be preserved, even though the data position would be. That inconsistency occurs when the content is created and after that the DOM nodes get changed. In such a case, the nodes are getting detached from the DOM and then reattached again in a different place. That's why they will not keep their scroll state, but ever since the component is not destroyed/recreated, it will still have its internal state. Some possible workarounds are: <br/> <ul><li>Resetting the DOM node's state, for example by wrapping it in an ngIf</li><li>Persisting the state, i.e. determining whether the element has been re-added to the DOM tree and resetting its scroll position manually</li></ul>
```

Comment:
Here's what I suggest:
| Scroll position of components that use igxForOf is not preserved in projected container |When component that uses `igxForOf` directive is in `<ng-content>` or other projected container its scrollbar position won't be preserved, even though the data position would be. That's because the DOM elements are detached and later re-attached to the DOM tree as a result losing the scrollbar scroll position.
Some possible workarounds are: <br/> <ul><li>Resetting the DOM node's state, for example by wrapping it in an `ngIf`. As a result the component will be destroyed and then re-created loosing all its state.</li><li>Persisting the state, i.e. determining whether the element has been re-added to the DOM tree and resetting its scroll position manually. For example using [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver).</li></ul>

Reply:
Done